sdk/core/azure-core-amqp/test/ut/claim_based_security_tests.cpp

```diff
@@ -47,6 +47,8 @@ namespace Azure { namespace Core { namespace Amqp { namespace Tests {
       GTEST_LOG_(INFO) << "CbsOperations" << CbsOperationResult::Invalid;
       GTEST_LOG_(INFO) << "CbsOperations" << CbsOperationResult::Failed;
       GTEST_LOG_(INFO) << "CbsOperations" << CbsOperationResult::InstanceClosed;
+      GTEST_LOG_(INFO) << "CbsOperations" << CbsOperationResult::Cancelled;
+      GTEST_LOG_(INFO) << "CbsOperations" << CbsOperationResult::Ok;
```

Comment:
What is the point of these lines in our test code? That it compiles? Do you want to compare the resulting string it actually puts to the stream instead?

Reply:
I think it would be better. And the coverage number would also mean more - that the code is actually verified, not just executed.

Reply:
I see this being repeated in several places, the comment stands for all of them.